sample-code/src/aiservice.ts

```diff
@@ -16,9 +13,8 @@ export async function chatCompletion(): Promise<string> {
     },
     'gpt-35-turbo'
   );
-  const assistantMessage = response.choices[0]
-    .message as OpenAiChatAssistantMessage;
-  return assistantMessage.content!;
+  const assistantMessage = response.getContent();
```

Comment:
You could also:
```ts
return response.getContent()!
```

Unless you wanted to. make explicit that an assistant message is returned